src/backend/rest.ts

```diff
@@ -94,6 +103,10 @@ export default class RestApiBackend implements Backend {
     } catch (error: any) {
       // grafana error handling is only showing status code and status message,
       // but we may have a more detailed error message
+      if (error.status === 404) {
+        throw new Error("REST-API is unavailable. Please make sure you have at least CheckMK 2.2.0 installed.")
```

Comment:
```suggestion
        throw new Error("REST API graph endpoints are unavailable. Please make sure you have at least Checkmk 2.2.0 installed.")
```
* docs use `REST API`
* docs use `Checkmk`
* REST API may be available, but just the graph endpoints may be missing
https://docs.checkmk.com/latest/en/rest_api.html